src/web/src/pages/signup.tsx

```diff
@@ -153,6 +153,14 @@ const useStyles = makeStyles((theme: Theme) =>
   })
 );
 
+enum SIGN_UP_STEPS {
```

Comment:
I'm adding an enum as required by @humphd (when we were working on the code before the 2.0 release.